src/controllers/Order.controller.ts

```diff
@@ -0,0 +1,144 @@
+import { Request, Response } from "express";
+import Cart from "../database/models/Cart.model";
+import Order from "../database/models/Order.model";
```

Comment:
I don't know if it's from personal preference, but abstracting the logic of communicating with the db to a separate folder like that of `Services`, would be much better easier to read. 🤷🏽‍♂️ 

e.g. writing a Order.service.ts and putting there any function that directly deals with the db, like creating a order, tracking an order ...